multiplication_table.py

```diff
@@ -1,14 +1,9 @@
-from sys import argv # import argment variable 
+argv = input('Number of ROWS and COLUMBS: ')
 
-script, rows, columns = argv # define rows and columns for the table and assign them to the argument variable
+script, rows, columns = argv, argv, argv # define rows and columns for the table and assign them to the argument variable
```

Comment:
The __input()__ statement is asking for two values, not three.  We should also deal gracefully with commas and spaces in the input: __rows, columns = argv.strip().replace(",", "").split()__